community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tftpl

```diff
@@ -55,8 +55,17 @@
 
   - name: Clones into installation directory
     ansible.builtin.command: git clone --branch {{ git_ref }} {{ git_url }} {{ install_dir }}
+    register: clone_res
+    failed_when: false
     when: lock_out.rc == 0
 
+  - name: Clone repository and set to specific commit
+    ansible.builtin.command: "{{ item }}"
+    with_items:
+      - git clone {{ git_url }} {{ install_dir }}
+      - git --git-dir={{ install_dir }}/.git reset --hard {{ git_ref }}
```

Comment:
I don't think we want to use `reset` here. Reset will move the branch that is currently checked out. For example if they have main checked out then main will get moved to the specific commit. 

I think what we want is a `git checkout <hash>` which will enter headless mode.

Reply:
Done, simplified since we can just do checkout during the original task and get rid of the new one.